com.kgurgul.cpuinfo.yml

```
@@ -0,0 +1,51 @@
+app-id: com.kgurgul.cpuinfo
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk17
+command: cpuinfo
+finish-args:
+  - --socket=x11
+  - --share=ipc
+  - --share=network
+  - --device=all
+  - --socket=ssh-auth
+modules:
+  - name: openjdk
+    buildsystem: simple
+    build-commands:
+      - /usr/lib/sdk/openjdk17/install.sh
+  - name: cpuinfo
+    buildsystem: simple
+    build-commands:
+      - install -D cpuinfo /app/bin/cpuinfo
+      - install -D cpuinfo.jar /app/share/cpuinfo.jar
+      - install -D com.kgurgul.cpuinfo.desktop /app/share/applications/com.kgurgul.cpuinfo.desktop
+      - install -D icon.png /app/share/icons/hicolor/256x256/apps/com.kgurgul.cpuinfo.png
+      - install -D com.kgurgul.cpuinfo.metainfo.xml /app/share/appdata/com.kgurgul.cpuinfo.metainfo.xml
+    sources:
+      - type: file
+        path: cpuinfo
+      - type: file
+        url: https://github.com/kamgurgul/cpu-info/releases/download/jvm-1.2.0/CPU-Info-linux-x64-1.2.0-release.jar
+        sha256: 8896c8440e3e73cb1cebe59344a2f7067189a5176fb673a4bbff3c3231b65482
+        only-arches: [x86_64]
+        dest-filename: cpuinfo.jar
+      - type: file
+        url: https://github.com/kamgurgul/cpu-info/releases/download/jvm-1.2.0/CPU-Info-linux-arm64-1.2.0-release.jar
+        sha256: ee51af1cb30bafbd9a4d30fa361c64f286dca517b59a475bb607195757536769
+        only-arches: [aarch64]
+        dest-filename: cpuinfo.jar
+      - type: file
+        url: https://raw.githubusercontent.com/kamgurgul/cpu-info/master/desktopApp/stores/flatpak/com.kgurgul.cpuinfo.desktop
```

```suggestion
        url: https://raw.githubusercontent.com/kamgurgul/cpu-info/a5a326d0da79de38be379f7deec7b571b70d2892/desktopApp/stores/flatpak/com.kgurgul.cpuinfo.desktop
```

Never use a branch in the URL. if sha256 changes the build will fail.